pkg/opensearch/ppl_response_parser.go

```diff
@@ -224,7 +226,12 @@ func getTimeSeriesResponseMeta(schema []client.FieldSchema) (responseMeta, error
 	if !found {
 		return responseMeta{}, errors.New("a valid time field type was not found in response")
 	}
-	return responseMeta{valueIndex: 1 - timeIndex, timeFieldIndex: timeIndex, timeFieldFormat: format}, nil
+
+	var valueIndex int
+	if timeIndex == 0 {
```

Comment:
Because of how the rest of the function logic works this actually gets the same value as the old code, but its a little easier to read